pkg/operator/controller/dns/controller_test.go

```diff
@@ -0,0 +1,119 @@
+package dns
+
+import (
+	"fmt"
+	"github.com/openshift/cluster-ingress-operator/pkg/dns"
+	"strings"
+
+	configv1 "github.com/openshift/api/config/v1"
+	operatorv1 "github.com/openshift/api/operator/v1"
+	iov1 "github.com/openshift/cluster-ingress-operator/pkg/api/v1"
+
+	"testing"
+)
+
+func TestPublishRecordToZones(t *testing.T) {
+	var tests = []struct {
+		name   string
+		record *iov1.DNSRecord
```

Comment:
`record` is always the same; how about omitting it here and defining it in the `for` loop?